codeql-workspace.yml

```diff
@@ -0,0 +1,29 @@
+provide:
+  - "*/ql/src/qlpack.yml"
+  - "*/ql/lib/qlpack.yml"
+  - "*/ql/test/qlpack.yml"
+  - "*/ql/examples/qlpack.yml"
+  - "*/ql/consistency-queries/qlpack.yml"
+  - "cpp/ql/test/query-tests/Security/CWE/CWE-190/semmle/tainted/qlpack.yml"
+  - "go/ql/config/legacy-support/qlpack.yml"
+  - "go/build/codeql-extractor-go/codeql-extractor.yml"
+  - "javascript/ql/experimental/adaptivethreatmodeling/lib/qlpack.yml"
+  # This pack is explicitly excluded from the workspace since we
+  # require consumers to use a version of this pack from the package cache.
```

Comment:
```suggestion
  # This pack is explicitly excluded from the workspace since most users
  # will want to use a version of this pack from the package cache. Internal
  # users can uncomment the following line and place a custom ML model
  # in the corresponding pack to test a custom ML model within their local
  # checkout.
```